internal/envconfig/envconfig.go

```diff
@@ -25,11 +25,15 @@ import (
 )
 
 const (
-	prefix          = "GRPC_GO_"
-	txtErrIgnoreStr = prefix + "IGNORE_TXT_ERRORS"
+	prefix                 = "GRPC_GO_"
+	txtErrIgnoreStr        = prefix + "IGNORE_TXT_ERRORS"
+	disableCompressorAdStr = prefix + "DISABLE_COMPRESSOR_ADVERTISEMENT"
 )
 
 var (
 	// TXTErrIgnore is set if TXT errors should be ignored ("GRPC_GO_IGNORE_TXT_ERRORS" is not "false").
 	TXTErrIgnore = !strings.EqualFold(os.Getenv(txtErrIgnoreStr), "false")
+	// DisableCompressorAd is set if registered compressor advertisement should
+	// be disabled ("GRPC_GO_DISABLE_COMPRESSOR_ADVERTISEMENT" is "true").
+	DisableCompressorAd = strings.EqualFold(os.Getenv(disableCompressorAdStr), "true")
```

Comment:
Let's go ahead and enable this by default (!=false).

Reply:
Compressor advertisement is enabled by default as `Disable != true`. But I have updated it to look like !=false
`GRPC_GO_ADVERTISE_COMPRESSORS != false`